includes/payment-methods/class-giropay-payment-method.php

```diff
@@ -23,6 +23,7 @@ public function __construct( $token_service ) {
 		parent::__construct( $token_service );
 		$this->stripe_id   = 'giropay';
 		$this->title       = 'Giropay';
+		$this->short_title = 'giropay';
```

Comment:
So I've seen that stylistically giropay has been displayed with a lowercase g elsewhere on the web, just wanted to confirm that was the intention here? Was this case the only need for the `short_title` field? I just wanted to get any background on the introduction of the short title field so that I use it properly going forward as well.

Reply:
I'm glad you brought this up! Originally, the CC payment method was labelled `Card` in the checkout gateway title, but `Credit card / debit card` as a payment method on the order complete page, so I made two separate titles. But ultimately, it's been decided to use the longer title for both fields, so I don't really need both fields anymore. I've seen Giropay stylised at `giropay` in quite a few places in our designs. @bborman22, do you think we should just use `giropay` as the title instead of capitalised `Giropay`? If so, then I'll just get rid of the `short_title` and use the main `title` for everything. 

The only other reason I can think to keep two titles could be for SEPA if, for example, we decided to display `SEPA direct debit` on the order complete page, but the shorter `SEPA` when used as a title. However, I've no idea whether that would be desirable or not.

Reply:
It seems like the lowercase vs uppercase is a bit open. All of the Stripe docs and giropay itself use lowercase, but in the Stripe Gateway plugin it defaults to uppercase. I prefer uppercase, but I suppose we should defer to the organization itself and use lowercase. And so if I'm understanding, the title was used on the order complete page and short title was for the radio select label primarily? If so, I think getting them all into one is good for now to avoid any confusion and can be addressed if needed in the future? But I can be easily convinced otherwise and don't feel too strongly about adding a short title here.

Reply:
I, too, prefer uppercase, because I'm kinda a fan of this little thing called **grammar**. But in the end, I agree with you! I think it doesn't make sense to keep this extra field right now and we can always add it back in somewhere down the line, if it turns out it might become useful. Just updated this in my last commit!